service.go

```diff
@@ -176,3 +205,49 @@ func storeOrgToCache(db *bolt.DB, cacheToBeWritten []org, wg *sync.WaitGroup) {
 	}
 
 }
+
+// HELPER METHODS
+
+func (s *orgServiceImpl) orgCount() (int, error) {
+	var count int
+	err := s.db.View(func(tx *bolt.Tx) error {
+		bucket := tx.Bucket([]byte(cacheBucket))
+		if bucket == nil {
+			return fmt.Errorf("Bucket %v not found!", cacheBucket)
+		}
+
+		count = bucket.Stats().KeyN
+		return nil
+	})
+
+	return count, err
+}
+
+func (s *orgServiceImpl) orgIds() ([]orgUUID, error) {
+	var uuidList []orgUUID
+	err := s.db.View(func(tx *bolt.Tx) error {
+		bucket := tx.Bucket([]byte(cacheBucket))
+		if bucket == nil {
+			return fmt.Errorf("Bucket %v not found!", cacheBucket)
+		}
+
+		log.Printf("List created, size = %d", bucket.Stats().KeyN)
```

Comment:
Is this a valuable printf statement. If you were supporting this would you know what a the "list" is?